ovirtbmc/ovirtbmc.py

```diff
@@ -43,29 +44,33 @@ def __init__(self, authdata, port, address, vm_name, cache_status, engine_fqdn,
         )
         self.vms_service = self.ovirt_conn.system_service().vms_service()
         self.vm = None
```

Comment:
I think it would make more sense to have `self.vm_id` instead of `self.vm` now

Reply:
Done